src/PlatformLinux.cpp

```diff
@@ -183,6 +183,24 @@ bool Platform::dirRemove(const std::string& path) {
 	return true;
 }
 
+std::string Platform::dirGetLocal() {
+    char abs_path[1024];
+    ssize_t len = readlink("/proc/self/exe", abs_path, 1024);
+    if(len < 0|| len >= 1024)  {
+        return "./";
+    }
+    //remove executable name from abs_path
+    char break_point = '/';
+    char break_string = '\0';
```

Comment:
These should be constants IMHO.  I'd also find them used as literals in the code below more readable.

Reply:
That's a good point